src/Pages/Search/Search.jsx

```diff
@@ -1,10 +1,46 @@
-import React from 'react'
-import './Search.css'
+import React, { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { getCategories } from "../../utils";
+import "./Search.css";
 
-const Search = () => {
+const Search = ({filter, setFilter}) => {
+  const [categories, setCategories] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+  useEffect(() => {
+    getCategories().then((data) => {
+      console.log(data)
```

Comment:
This console.log should be removed

Reply:
In fact, Patrik's original comment still needs actioning. This line should be setIsLoading(true) or it won't be reset when filter is changed.